weave-js/src/components/PagePanelComponents/Home/Browse3/views/Leaderboard/LeaderboardGrid.tsx

```diff
@@ -111,8 +121,23 @@ export const LeaderboardGrid: React.FC<LeaderboardGridProps> = ({
         minWidth: 150,
         flex: 1,
         renderCell: (params: GridRenderCellParams) => {
+          let isOp = false;
+          try {
+            isOp =
+              Object.values(
+                Object.values(
+                  Object.values(
+                    (params.row as RowData).modelGroup.datasetGroups
+                  )[0].scorerGroups
+                )[0].metricPathGroups
+              )[0][0].modelType === 'op';
```

Comment:
i think we should make helpers for these deeply nested leaf value getters

Reply:
Ack - fixed in next commit